Aquality.Appium.Mobile.Template/Aquality.Appium.Mobile.Template/Screens/Android/Alert.cs

```diff
@@ -0,0 +1,16 @@
+﻿using Aquality.Appium.Mobile.Screens;
+using OpenQA.Selenium;
+
+namespace Aquality.Appium.Mobile.Template.Screens.Android
+{
+    public class Alert : AndroidScreen, Interfaces.IAlert
```

Comment:
using Interfaces namespace?

Reply:
There is a conflict with Selenium.IAlert interface